nixos/modules/services/misc/gitlab.nix

```diff
@@ -633,6 +669,22 @@ in {
     # Use postfix to send out mails.
     services.postfix.enable = mkDefault true;
 
+    # Enable Docker Registry, if GitLab-Container Registry is enabled
+    services.dockerRegistry = optionalAttrs cfg.registry.enable {
+      enable = cfg.registry.enable;
+      listenAddress = "127.0.0.1";
+      port = 5000;
+      enableDelete = true; # This must be true, otherwise GitLab won't manage it correctly
+      extraConfig = {
+        REGISTRY_LOG_LEVEL = "info";
```

Comment:
is this not the default?

Reply:
Deletion is not the default. When disabled, GitLab will throw errors, because it cannot fully delete projects.

Reply:
I was referring to the log level, at that line.